bitbucketclient.py

```diff
@@ -0,0 +1,27 @@
+import requests
+import json
+
+
+def get_commits_bitbucket(reponame, since_date):
```

Comment:
Can you create new class ``BitbucketClient`` with getCommits(reponame, since_date) method? bitbucketclient module can contain more methods in future and it is easier to replace object with another object with the same interface. I would like to have an instance that can be injected to other objects working with bitbucket repositories.